packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1574,3 +1574,22 @@ class SliverCrossAxisGroup extends MultiChildRenderObjectWidget {
     return RenderSliverCrossAxisGroup();
   }
 }
+
+/// A sliver that places multiple sliver children in a linear array along
+/// the main axis, one after another.
+///
+/// For pinned sliver children, the behavior is that the pinned sliver should
+/// scroll up after all of the main content has been scrolled through.
```

Comment:
Can you add a `See also` for SliverCrossAxisGroup? And then add one for the cross axis group that refers back here?

Reply:
Done!